src/server/service/quoteEvent.ts

```diff
@@ -102,6 +102,24 @@ function createQuoteEmbed(quoteMessage: Message) {
   const quoteChannelId = quoteMessage.channel.id;
   const quoteUserAvatar = quoteUser.avatarURL();
 
+  let quoteMessageType: string = quoteMessage.type;
+  // システムメッセージ等、MessageQuoteが引用できないものは除外
+  // 参考: https://discord.js.org/#/docs/discord.js/stable/typedef/MessageType
+  switch (quoteMessageType) {
```

Comment:
既に詳しく型付けされている `quoteMessage.type` を `string` に変換しないでください. プログラムミスの原因になります
```suggestion
  let quoteMessageType: string;
  // システムメッセージ等、MessageQuoteが引用できないものは除外
  // 参考: https://discord.js.org/#/docs/discord.js/stable/typedef/MessageType
  switch (quoteMessage.type) {
```